packages/augur-ui/src/modules/orders/actions/load-market-orderbook.ts

```diff
@@ -5,24 +5,7 @@ import { Action } from 'redux';
 import { augurSdk } from 'services/augursdk';
 import { AppState } from 'appStore';
 import { Getters } from '@augurproject/sdk';
-
-export const UPDATE_ORDER_BOOK = 'UPDATE_ORDER_BOOK';
-export const CLEAR_ORDER_BOOK = 'CLEAR_ORDER_BOOK';
-
-export const updateOrderBook = (
-  marketId: string,
-  orderBook: Getters.Markets.MarketOrderBook
-) => ({
-  type: UPDATE_ORDER_BOOK,
-  data: {
-    marketId,
-    orderBook,
-  },
-});
-
-export const clearOrderBook = () => ({
-  type: CLEAR_ORDER_BOOK,
-});
+import { useMarketsStore, Markets } from 'modules/markets/store/markets';
 
 export const loadMarketOrderBook = (
```

Comment:
Ideally we wanna get rid of this entire action file -- maybe move this one function to a helper?